project-base/storefront/helpers/isClient.ts

```diff
@@ -0,0 +1,6 @@
+export const isClient = (): boolean =>
+    !!(
+        typeof window !== 'undefined' &&
+        typeof window.document === 'object' &&
+        typeof window.document.createElement === 'function'
+    );
```

Comment:
Hmm, actually one more think which I am wondering about. Couldn't this be simple variable? 🤔

```suggestion
export const isClient = !!(
        typeof window !== 'undefined' &&
        typeof window.document === 'object' &&
        typeof window.document.createElement === 'function'
    );
```

Reply:
It can, nice catch.